flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -2012,6 +2012,9 @@
 	/** The environment variable name which contains the location of the plugins folder. */
 	public static final String ENV_FLINK_PLUGINS_DIR = "FLINK_PLUGINS_DIR";
 
+	/** The default Flink plugins directory is none has been specified via {@link #ENV_FLINK_PLUGINS_DIR}. */
```

Comment:
Is it a typo? is -> if
/** The default Flink plugins directory if none has been specified via {@link #ENV_FLINK_PLUGINS_DIR}. */

Reply:
It's a typo.

Reply:
Yes it's a typo. Will correct it.